server/routes/barcode/pdf/selectEnvelopeSize.spec.ts

```diff
@@ -0,0 +1,28 @@
+import SuperTestWrapper from '../../fixtures/SuperTestWrapper'
+import assertThat from '../../fixtures/supertest-assertions'
+
+describe('Select envelope size', () => {
+  const superTest = new SuperTestWrapper()
+
+  beforeEach(async () => {
```

Comment:
I've added a test that requires the user to be authenticated. The test spec itself could do with some more functional tests adding (ie. submitting the envelope size etc), but I wanted to make sure we could send requests as an authenticated user